pkg/builder/slurm_builder.go

```diff
@@ -532,16 +556,32 @@ func (b *slurmBuilder) buildInitEntrypointScript(jobName string) (string, error)
 		SlurmMemPerGPU:      getValueOrEmpty(b.memPerGPU),
 		SlurmMemPerNode:     getValueOrEmpty(b.totalMemPerNode),
 		SlurmGPUs:           getValueOrEmpty(b.totalGpus),
-		SlurmNTasks:         nTasks,
-		SlurmNTasksPerNode:  nTasks,
-		SlurmNProcs:         nTasks,
-		SlurmNNodes:         nodes,
-		SlurmSubmitDir:      slurmScriptsPath,
-		SlurmJobNodeList:    strings.Join(nodeList, ","),
-		SlurmJobFirstNode:   nodeList[0],
-
-		FirstNodeIP:               b.firstNodeIP,
-		FirstNodeIPTimeoutSeconds: int32(b.firstNodeIPTimeout.Seconds()),
+
+		SlurmNTasks:        nTasks,
+		SlurmNTasksPerNode: nTasksPerNode,
+		SlurmJobNumNodes:   nNodes,
+		SlurmJobNodeList:   strings.Join(nodeList, ","),
+		SlurmJobFirstNode:  nodeList[0],
+		SlurmFirstJobID:    slurmFirstJobID,
+		// SlurmTasksPerNode:  nTasks, // TODO: Implement buildTasksPerNode() builder according to https://slurm.schedmd.com/sbatch.html#OPT_SLURM_TASKS_PER_NODE.
```

Comment:
Currently, these variables are not being calculated correctly. I would propose fixing them in a separate PR.